doc/manual/src/glossary.md

```diff
@@ -215,6 +215,9 @@
 
   [output path]: #gloss-output-path
 
+- [output closure]{#gloss-output-closure}\
+  The [closure] of an [output path]. It only contains what is [reachable] from the output.
```

Comment:
Maybe add an editing note, to add to the relevance.

```suggestion
  The [closure] of an [output path]. It only contains what is [reachable] from the output.
  <!-- It is not the closure over the outputOf relation, which we'll leave out because it's a "dynamic derivations" thought that needs some deep thinking to even come up with, certainly as long as people aren't used to RFC 92 -->
```

Reply:
Probably not necessary here, but I understand what you mean.